server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -292,3 +292,31 @@ watermark_module_opened=false
 watermark_secret_key=IoTDB*2019@Beijing
 watermark_bit_string=100101110100
 watermark_method=GroupBasedLSBMethod(embed_row_cycle=2,embed_lsb_num=5)
+
+
+####################
+### Configurations for creating schema automatically
+####################
+
+# Is automatically creating schema enable
+enable_auto_create_schema=true
+
+# Storage group level when creating schema automatically is enable
+# The default is 2
+auto_storage_group_level=2
+
+# Boolean encoding when creating schema automatically is enable
+# The default is RLE
+auto_boolean_encoding=RLE
+
+# Long encoding when creating schema automatically is enable
+# The default is RLE
+auto_long_encoding=RLE
+
+# Double encoding when creating schema automatically is enable
+# The default is GORILLA
+auto_double_encoding=GORILLA
+
+# String encoding when creating schema automatically is enable
+# The default is PLAIN
+auto_string_encoding=PLAIN
```

Comment:
I wish you could rename these to `default_xxx_encoding` and one day we will not need to specify an encoding in  "create timeseries" statements.